src/components/PopupConfirm.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+  }
+
+  setEventListeners() {
+    this._popupForm.addEventListener("submit", (e) => {
+      e.preventDefault();
+      this._handleFormSubmit();
+      super.close();
+    });
+    super.setEventListeners();
+  }
+
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.